python/featomic/featomic/clebsch_gordan/_coefficients.py

```diff
@@ -197,18 +197,15 @@ def _build_dense_cg_coeff_dict(
                     device=device,
                     dtype=complex_like.dtype,
                 )
+                # TODO: comment to explain what is happening here and the shapes
+                # involved
+                real_cg = (_dispatch.permute(complex_cg, [2, 1, 0]) @ r2c[l1]).swapaxes(
```

Comment:
please use temporaries instead of doing everything in a single line, it is already hard enough to understand =)

Reply:
Fair enough!